spec/order_list_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'order_list'
+
+describe OrderList do
+
+  let(:pizza){double(:pizza, name: :pizza, price: 1)}
+  let(:chicken){double(:chicken, name: :chicken, price: 2)}
+  let(:dish_list){double(:dish_list, price_list: {pizza: 1, chicken: 2}, dishes: [pizza, chicken])}
+  subject(:order_list) {described_class.new(dish_list)}
+  before(:example){allow(order_list.client.messages).to receive(:create)}
+
+  describe '#price_list' do
+    it 'gives a list with the price of each dish' do
+      expect(dish_list).to receive(:price_list)
+      order_list.price_list
+    end
+  end
+
+  describe '#place_order' do
+
+    context 'a dish selected is unavailable' do
+      it 'raises an error' do
+        expect{order_list.place_order(apple: 2)}
+          .to raise_error "An ordered dish is unavailable"
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver. (https://github.com/bbatsov/ruby-style-guide#consistent-multi-line-chains)